github/ci/prow/files/jobs/kubevirt.github.io/kubevirt-github-io-postsubmits.yaml

```diff
@@ -0,0 +1,14 @@
+postsubmits:
+  kubevirt/kubevirt.github.io:
+    - name: kubevirt-io-postsubmit-link-checker
+      decorate: true
+      spec:
+        containers:
+          - image: docker.io/library/ruby
+            env:
+              - name: NOKOGIRI_USE_SYSTEM_LIBRARIES
+                value: "true"
+            command: ["/bin/sh", "-c"]
+            args: ["/usr/local/bin/bundle install && bundle exec rake"]
+            securityContext:
+              privileged: true
```

Comment:
Why do you need a privileged pod?

Reply:
I think I dont need it, let me ensure about it

Reply:
great.

Reply:
Oh, I don't think that you need this job at all. Tide uses a merge pool which reruns tests if master changed since the last ci run on merge. So there is no difference between a presubmit job from a merge or a postsubmit directly afterwards on master.

Reply:
yeah, maybe I could erase for now the postsubmit, until we get more functionallity checks. what do you think?

Reply:
Yes, it is just not needed in my opinion right now.

Reply:
Postsubmit removed :)